### PR TITLE
TF-2903 Handling of 3D links

### PR DESCRIPTION
**Pull request by dab246**
## Issue

#2903 

## Root cause

- Unstandardized url links starting with `www` in the `href` attribute of the `a` tag in html content

## Resolved

- Handle 3D links:

https://github.com/linagora/tmail-flutter/assets/80730648/42b4236a-e0f6-46cd-b513-4828d3fe4a91

- Handle click mailto link:

https://github.com/linagora/tmail-flutter/assets/80730648/57a99028-f0dc-4b45-b7c7-be421f5466d1

### CODE REVIEW

**Comment by dab246** [comment]
Rebase to the `refactor` is complete

File: core/lib/presentation/utils/html_transformer/dom/sanitize_tag_a_in_html_transformers.dart [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import 'package:core/data/network/dio_client.dart';
+import 'package:core/presentation/utils/html_transformer/base/dom_transformer.dart';
+import 'package:core/presentation/utils/html_transformer/sanitize_url.dart';
+import 'package:core/utils/html/html_template.dart';
+import 'package:html/dom.dart';
+
+class SanitizeTagAInHtmlTransformer extends DomTransformer {
```
Comment (tddang-linagora) [inline core/lib/presentation/utils/html_transformer/dom/sanitize_tag_a_in_html_transformers.dart:7]:
```suggestion
class SanitizeAnchorTagInHtmlTransformer extends DomTransformer {
```

Comment (tddang-linagora) [reply core/lib/presentation/utils/html_transformer/dom/sanitize_tag_a_in_html_transformers.dart:7]: Let's go with `SanitizeHyperLinkTagInHtmlTransformer`

Comment (dab246) [reply core/lib/presentation/utils/html_transformer/dom/sanitize_tag_a_in_html_transformers.dart]: done

**Review by tddang-linagora — COMMENTED** [review]
- [x] Tested on Web

**Review by tddang-linagora — APPROVED** [review]

**Review by tddang-linagora — APPROVED** [review]
- [x] Tested on Web

**Comment by hoangdat** [comment]
@dab246 
- [ ] please rebase

**Review by hoangdat — APPROVED** [review]

**Pull request merged**